p2p/module.go

```diff
@@ -0,0 +1,53 @@
+package pre_p2p
+
+import (
+	"log"
+	"pocket/shared/config"
+	"pocket/shared/modules"
+
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+var _ modules.NetworkModule = &networkModule{}
+
+type networkModule struct {
+	pocketBus modules.Bus
```

Comment:
Let's try to leave out the prefix `pocket` from things when possible

Reply:
👍  Replaced all instances of `pocketBus` with `bus`.